linera-chain/src/chain.rs

```diff
@@ -484,13 +484,31 @@ where
 
         balance.try_add_assign(credit)?;
         Self::sub_assign_fees(balance, pricing.certificate_price())?;
-        Self::sub_assign_fees(balance, pricing.storage_price(&block.incoming_messages)?)?;
-        Self::sub_assign_fees(balance, pricing.storage_price(&block.operations)?)?;
+        Self::sub_assign_fees(
+            balance,
+            pricing.storage_bytes_write_price(&block.incoming_messages)?,
+        )?;
+        Self::sub_assign_fees(
+            balance,
+            pricing.storage_bytes_write_price(&block.operations)?,
+        )?;
```

Comment:
I wonder whether we should keep these price categories separate, and just rename `storage_price` to `block_storage_price` or something?

An application's execution state will probably more often be read and overwritten on average, and sometimes deleted. On the other hand, there's no limit for how long it has to be kept.

Stored blocks can never be overwritten, but they might get archived after some time.

Reply:
The block_storage price is something we want to implement afterward.